prow/repoowners/repoowners.go

```diff
@@ -682,6 +694,7 @@ func NormLogins(logins []string) sets.String {
 
 var defaultDirOptions = dirOptions{}
```

Comment:
as long as we are near here:
`var defaultDirOptions dirOptions`

https://dave.cheney.net/practical-go/presentations/qcon-china.html#_use_a_consistent_declaration_style

Reply:
Agreed, I changed this.